### ISSUE TITLE
Abnormal neck righting reflex

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6422]
**Preferred term label:** Abnormal neck righting reflex

**Synonyms**

**Definition (free text, please give PubMed ID)**  Lack of or reduced movement of the trunk in the direction the head is turned.

**Parent term (use hpo.jax.org/app)** Abnormal reflex HP:0031826

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term
